Тултип

ksanster opened the pull request.

Commit:
Add solution for 05-lection3/01-tooltip

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 05-lection3/01-tooltip/tooltip.css:
```diff
@@ -0,0 +1,43 @@
+.tooltip {
+    display: inline-block;
```
вот здесь не забываем про position: relative

ksanster replied:
`position:relative` для того, чтобы tooltip__bubble позиционировался относительно блока tooltip?
После этого изменения текст начинает переноситься по строкам - не совсем понимаю почему. Или не понимаю, почему раньше не переносился, что, впрочем, одно и то же.
Предполагаю, что до изменения ширина bubble определялась внешними ограничениями, а после стала определяться только контентом, но понимания нет.
Можно, конечно, закостылить через min-width или `white-space: nowrap;`, в зависимости от ожидаемого поведения (я выбрал второе)

GSemikozov replied:
до этого ближайшим родителем относительно которого позиционировались был body, ширина которого больше чем ширина бабла, так что текст там не переносился, а когда начали позиционировать относительно тултипа - ситуация поменялась

GSemikozov replied:
стратегия была выбрана совершенно правильно, как и предположения тоже были правильными

Commit:
Update from upstream
Merge branch 'master' of github.com:js-tasks-ru/htmlcss-20231027_ksanster into tooltip

Commit:
Fix styles
